packages/jetpack-ai-calypso/package.json

```diff
@@ -0,0 +1,71 @@
+{
+	"name": "@automattic/jetpack-ai-calypso",
+	"version": "1.0.0",
+	"description": "Jetpack AI utilities available to Calypso",
+	"calypso:src": "src/index.js",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"sideEffects": [
+		"*.css",
+		"*.scss"
+	],
+	"keywords": [
+		"wordpress"
```

Comment:
Probably could be altered a bit to contain for example `jetpack`, `ai` and `utilities`.